src/condor_utils/docker-api.cpp

```diff
@@ -285,11 +285,30 @@ int DockerAPI::createContainer(
 	jobAd.LookupString(ATTR_DOCKER_NETWORK_TYPE, networkType);
 	if (networkType == "host") {
 		runArgs.AppendArg("--network=host");
-	}
-	if (networkType == "none") {
+	} else if (networkType == "none") {
 		runArgs.AppendArg("--network=none");
+	} else if (!networkType.empty()) {
+		// We always allow network type "none" or "host",
+		// as "none" is the safest, and "host" is like a vanilla job
+		// But others must be allowed by the admin
+		std::string network_param;
+		param_eval_string(network_param, "DOCKER_NETWORKS", "", &jobAd, &machineAd);
+
```

Comment:
You should not param this here,  the expression has been assigned as an attribute in the machineAd.